### DISCUSSION TITLE
Method mutator

### DISCUSSION THREAD

**Discussion by richardwerkman** [stryker-mutator/stryker-net#209] [Ideas]
Some methods (other than Linq methods) can be mutated into their counterparts.

For example:
``` csharp
list.IndexOf(4);
```
can be mutated into:
``` csharp
list.LastIndexOf(4);
```
I would like to discuss what kind of methods could be mutated by this mutator. Removing methods is also an option.

**Comment by mdaniyalkhann** [comment]
https://github.com/MilleBo/Unima/blob/master/src/Unima.Core/Creator/Mutators/MethodCallMutator.cs

**Comment by lukasz-pekala** [comment]
Link mentioned by @mdaniyalkhann is dead (404)
@Mobrockers perhaps you should label it again with "information needed" ?

**Comment by rouke-broersma** [comment]
It was simply renamed, file can be found here: https://github.com/Testura/Testura.Mutation/blob/master/src/Testura.Mutation.Core/Creator/Mutators/MethodCallMutator.cs that said the information there is not very useful for our intended use case.

Let's talk about some possible types where we could introduce these kinds of mutations.

Math probably has a lot of methods that have an inversion.
List, like @richardwerkman suggested
